pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -72,18 +91,15 @@ func checkHyperVInstalled() (bool, error) {
 	return true, nil
 }
 
-//
-func fixHyperVInstalled() (bool, error) {
-	enableHyperVCommand := `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`
-	_, _, err := powershell.ExecuteAsAdmin("enable Hyper-V", enableHyperVCommand)
+func fixHyperVServiceRunning() (bool, error) {
+	enableVmmsService := `Set-Service -Name vmms -StartupType Automatic; Set-Service -Name vmms -Status Running -PassThru`
```

Comment:
When would this be disabled?

Reply:
Why disabled? Maybe the same as 'Why do people disable libvirt'? Maybe disabled by the person because it consumes tons of memory when used? Perhaps disabled because of another hypervisor? We can't tell... but when started it fails with a clear error.

Reply:
Let me ask the question differently :) Are there systems where hyperv is going to be installed but disabled without the user doing anything? Or will it only be disabled when the user intentionally disabled it?

Reply:
Since the merging of this is going to be delayed, I'd still mention in the commit log that this situation should only happen when uninstalling hyperv, or when using restore checkpoints, ... and that it will not happen without user intervention.